### PR TITLE
feat: add expandable actions + light up for tb action + some more community made graphics (and alt texts)

### PR DESCRIPTION
**Pull request by YummyOreo**

### CODE REVIEW

File: docs.css [hunk 1]
```diff
@@ -71,3 +71,33 @@ img.letter {
...
+
+h3:hover>.header-copy {
+	opacity: 1;
+}
+
+.header-copy:hover::after {
+	content: url("./img/link-hover.svg");
+	opacity: 1;
+	cursor: pointer;
+}
```
Comment (YummyOreo) [inline docs.css:96]: is there a way to color it without having two different svg elements?

File: docs.css [hunk 2]
```diff
@@ -71,3 +71,33 @@ img.letter {
 		margin: 0.125em 0.125em;
 	}
 }
+
+/* header links */
+.header-copy {
+	height: 0.8em;
+	width: 1em;
+	position: absolute;
+	transform: translate(-1.5em, 0);
```
Comment (YummyOreo) [inline docs.css:80]: test this on different screen sizes (heights and widths)

Comment (YummyOreo) [reply docs.css:80]: possibly move it to the right for mobile support

File: action.html [hunk 3]
```diff
@@ -22,10 +22,12 @@
 		<link href="https://fonts.googleapis.com/css2?family=Atkinson+Hyperlegible:ital,wght@0,400;0,700;1,400;1,700&display=swap"
 		rel="stylesheet" />
 		<link rel="stylesheet" href="style.css" />
+		<link rel="stylesheet" href="actions.css" />
```
Comment (YummyOreo) [inline action.html:25]: is this needed/used?

File: action.html [hunk 4]
```diff
@@ -126,10 +143,56 @@ <h2 id="contact">Contact <span class="red">Danaher and Cepheid</span></h2>
...
+							<noscript>
+								<p class="smaller" style="margin-bottom: 0;"><em>Please enable javascript to use the copy buttons.</em></p>
+							</noscript>
+							<button class="button" id="copy-light">Copy template</button>
+							<a href="https://docs.google.com/document/d/e/2PACX-1vSDHBl8p3AN1jZOx4fmvwvtk3XNrbMh0TkHsgm0MxinwigHCGHp79_-sdrdW0v6AA/pub" target="_blank" class="button button-subtle">View template</a>
+						</p>
+					</div>
+					<div class="action">
+						<p class="larger">
+							<img src="img/megaphone.svg" alt="light icon" class="action-icon" /><strong><span class="red">Spread</span></strong> the Word About <span class="red">TB</span>
```
Comment (YummyOreo) [inline action.html:164]: alt text needs to be updated

File: action.html [hunk 5]
```diff
@@ -126,10 +143,56 @@ <h2 id="contact">Contact <span class="red">Danaher and Cepheid</span></h2>
...
+						<p class="smaller">
+							Educate and inform others on the worldwide impact of tuberculosis - the world's deadliest infectious disease, but one many people assume is a thing of the past. Learn more on our resource page below.
+						</p>
+						<p class="smaller">
+						<a href="resources.html" class="button" target="_blank">Resource page</a>
+						</p>
+					</div>
+					<div class="action">
+						<p class="larger">
+							<img src="img/gavel.svg" alt="light icon" class="action-icon" /><strong><span class="red">Support</span></strong> Legislative and Government Action
```
Comment (YummyOreo) [inline action.html:175]: alt text

File: action-page.js [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+It is an honor to invite ________ to join the global movement and light up a city landmark, building, or clock in RED on 24 March 2024, to bring attention to the devastating impact of TB.
+
+The Stop TB Partnership launched the “Light Up for TB” campaign for the first time in 2017, and we have attracted broad participation from over 60 cities in previous campaigns.
+
+Now more than ever, the world community must come together to end this terrible disease.
+
+We hope that _________ will show your support by joining the initiative for World TB Day 2024 and Light Up for TB. Time is running out – we must invest in ending TB and saving lives.
+
+Yours sincerely,
+[YOUR NAME] `)
```
Comment (YummyOreo) [inline action-page.js:44]: put this all into a const

**Review by Nolram12345 — APPROVED** [review]

**Comment by YummyOreo** [comment]
Please squash instead of merge to keep history clean

**Review by devoeas — APPROVED** [review]

**Pull request merged**